docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+The mark increment operation in the example marked 7,726,701 objects and scanned 7,584,109 of these marked objects. The 7,584,109 of scanned objects take up 203.5 MB (213445656 B) of memory. The number of scanned objects is less than the number of marked objects because only objects that have children require scanning. Also, the scanning part of the marking operation might be interrupted by the garbage collector if a trigger threshold for a partial cycle or global cycle is reached during the marking operation.
+
+The STW `global mark phase` subincrement ends, as recorded by `<gc-end>`, which records a snapshot of the memory status in the heap in a similar way to `<gc-start>`. 
+
+```xml
+<gc-end id="1158" type="global mark phase" contextid="1154" durationms="123.139" usertimems="977.851" systemtimems="0.000" stalltimems="1.453" timestamp="2021-02-26T11:17:25.157" activeThreads="8">
+  <mem-info id="1159" free="1442840576" total="4294967296" percent="33">
+    <remembered-set count="3263968" freebytes="158648448" totalbytes="171704320" percent="92" regionsoverflowed="0" regionsstable="0" regionsrebuilding="133"/>
+  </mem-info>
+</gc-end>
+```
+
+Comparing the snapshot at the beginning and end of this STW `global mark phase` subincrement shows that:
+
+- The marking operation has increased the `count` value of the `<remembered-set>` by 1,066,080 cards (from 2,197,888 to 3,263,968)
+- The number of overflow regions has reduced from three to zero. This GMP increment has rebuilt all overflow regions.
+- As expected of a global mark cycle's operations, there is no change in the amount of free memory available, which is 1376 MB.
+
+The beginning of the second part of the global *mark* phase increment, the GMP work packet procesing subincrememt, is recorded by `<concurrent-start>`. The child element `<concurrent-mark-start>` records the scan target of this subincrement as 242.74 MB (254,532,672 B).
+
+```xml
+<concurrent-start id="1160" type="GMP work packet processing" contextid="1154" timestamp="2021-02-26T11:17:25.157">
+  <concurrent-mark-start scanTarget="254532672" />
+</concurrent-start>
+```
+
+Now that the STW global mark phase sub-increment is complete, application threads are restarted. 
+
+```xml
+<exclusive-end id="1161" timestamp="2021-02-26T11:17:25.157" durationms="123.936" />
+```
+
+The `GMP work packet processing` sub-increment continues to run concurrently. The end of this subincrement's operations are recorded using the `<concurrent-end>` element. 
+
+```xml
+<concurrent-end id="1162" type="GMP work packet processing" contextid="1154" timestamp="2021-02-26T11:17:25.469" terminationReason="Work target met">
+<gc-op id="1163" type="mark increment" timems="311.867" contextid="1154" timestamp="2021-02-26T11:17:25.469">
+  <trace-info scanbytes="254708852" />
+</gc-op>
+</concurrent-end>
+```
+
+The child element `<trace-info>` shows that the processing has scanned 242.91 MB (254,708,852 B), which exceeds the 108.25 MB scan target.
```

Comment:
actually the actual work of 242.91MB is pretty close to the target of 242.74 MB

Reply:
Modified to "which slightly exceeds the 108.25 MB scan target."